test/functionalTest/cases/0987_put_attributes/put_entity_error.test

```diff
@@ -55,8 +55,9 @@ echo
 
 echo "03. Create E1"
 echo "============="
+# FIXME P10: This creation is not legal! Maybe related with issue #1682
```

Comment:
I have spotted this (and other similar ones) when modifying the .test... The point is that `POST /v2/entities/E1` can be used to append or update entity attributes, but not for creating new attributes. I'd review (and eventually fix in a PR) after this PR.

NTC